### PR TITLE
Correct typos for most of section 1 docs

### PR DESCRIPTION
**Pull request by wolf99**
I saw "then" in place of "than" and just couldn't keep reading without fixing it.

### CODE REVIEW

File: _docs/01-advanced-features.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ In each captured folder, we obtain the file `icfg.dot`, which is the graphical r
 `call_graph.dot`, that is the graphical representation of the call graph.
 
 
-Moreover, we obtain an html page for each captured file inside `infer-out/captured`. This html file contains the source file. In each line of the file there are links to the nodes of the control flow graph that correspond to that line of code. So one can see what the translation looks like. Moreover, when you click on those links you can see details of the symbolic execution of that particular node. If the option `--no_test` is also passed to `infer`, then the page pointed to from the nodes contains the printout of the whole symbolic execution.
+Moreover, we obtain an HTML page for each captured file inside `infer-out/captured`. This HTML file contains the source file. In each line of the file there are links to the nodes of the control flow graph that correspond to that line of code. So one can see what the translation looks like. Moreover, when you click on those links you can see details of the symbolic execution of that particular node. If the option `--no_test` is also passed to `infer`, then the page pointed to from the nodes contains the printout of the whole symbolic execution.
```
Comment (wolf99) [inline _docs/01-advanced-features.md]: Here I changed the html to HTML (x2). I'm not sure why the diff isn't showing the specific change. I notice other changes I've made have this issue so I'll add review notes in each case to point out the change

File: _docs/01-linters.md [hunk 2]
```diff
@@ -527,11 +527,11 @@ DEFINE-CHECKER ENUM_CONSTANTS = {
 
 <a name="info_message">**AST info in messages**</a>
 
-When you write the message of your rule, you may want to specify which particular ast items were involved in the issue, such as a type or a variable name. We have a mechanism for that, we specified a few placeholders that can be used in rules with the syntax `%placeholder%` and it will be substituted by the correct ast info. At the moment we have `%type%`, `%child_type%` and `%name%` that print the type of the node, the type of the node's child, and a string representation of the node, respectively. As with predicates, we can add more as needed.
+When you write the message of your rule, you may want to specify which particular AST items were involved in the issue, such as a type or a variable name. We have a mechanism for that, we specified a few placeholders that can be used in rules with the syntax `%placeholder%` and it will be substituted by the correct AST info. At the moment we have `%type%`, `%child_type%` and `%name%` that print the type of the node, the type of the node's child, and a string representation of the node, respectively. As with predicates, we can add more as needed.
```
Comment (wolf99) [inline _docs/01-linters.md]: Capitalise ast to AST

File: _docs/01-linters.md [hunk 3]
```diff
@@ -559,7 +559,7 @@ Moreover, the bug can be found in the file `infer-out/report.json` where `infer-
...
+If there are syntax errors or other parsing errors with your al file, you will get an error message when testing the rule, remember to use `linters-developer-mode` when you are developing a rule. If the rule gets parsed but still doesn't behave as you expect, you can debug it, by adding the following line to a test source file in the line where you want to debug the rule: `//INFER_BREAKPOINT`. Then run infer again in linters developer mode, and it will stop the execution of the linter on the line of the breakpoint. Then you can follow the execution step by step. It shows the current formula that is being evaluated, and the current part of the AST that is being checked. A red node means that the formula failed, a green node means that it succeeded.
```
Comment (wolf99) [inline _docs/01-linters.md]: `in the line of the breakpoint` => `on the line of the breakpoint`

**Comment by jvillard** [comment]
Thanks @wolf99! And thanks the review notes, it was painful to check the diff by hand.

**Pull request merged**